django_project/monitor/models.py

```diff
@@ -55,6 +60,12 @@ class Sites(models.Model):
     river_cat = models.CharField(max_length=5, choices=RIVER_CATS, blank=True)
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     time_stamp = models.DateTimeField(auto_now=True)
+    assessment_id = models.ForeignKey(Assessment, on_delete=models.CASCADE,blank=True,null=True)
+    miniSass_score = models.FloatField(blank=True,null=True)
+    miniSass_ML_score = models.FloatField(blank=True,null=True)
```

Comment:
```suggestion
    minisass_ML_score = models.FloatField(blank=True,null=True)
```

Please avoid using camelcase in strings for Python scripts.